File: sprint4/JS/lesson1code3.js
const fruits = ["яблоко", "яблоко", "груша", "яблоко", "слива"];
const fruitCount = new Map();
for (const fruit of fruits) {
    if (!fruitCount.has(fruit)) {
        fruitCount.set(fruit, 0);
    }
    fruitCount.set(fruit, fruitCount.get(fruit) + 1);
}

File: sprint4/go/lesson1code3.go
package main

import "fmt"

func main() {
    fruits := []string{"яблоко", "яблоко", "груша", "яблоко", "слива"}
    fruitCount := make(map[string]int)
    for _, fruit := range fruits {
        if _, ok := fruitCount[fruit]; !ok {
            fruitCount[fruit] = 0
        }
        fruitCount[fruit]++
    }

    for fruit, count := range fruitCount {
        fmt.Printf("%v: %v\n", fruit, count)
    }
}
